scss/_patterns_grid-8.scss

```diff
@@ -82,7 +82,7 @@
 
   // on medium/tablet screens, small and large grid class names span full width (to match block level element behaviour)
   %grid-8-span-on-tablet {
-    @media (min-width: $threshold-4-6-col) {
+    @media (min-width: $threshold-4-small-4-med-col) {
```

Comment:
these threshold queries were previously using thresholds for the legacy grid; i've introduced new variables for the new grid and bound them to the existing values